src/utils/users.ts

```diff
@@ -0,0 +1,16 @@
+import { createUser, getUserById } from '../db/users';
```

Comment:
Why extract this out to its own file?
Is it just because it doesn't actually do the DB interaction directly?
It kind of makes sense especially if we think we might have other user-related utilities later..but it might also be a bit premature.